ts/src/transports/fetch.ts

```diff
@@ -44,12 +45,13 @@ class Fetch implements Transport {
       });
   }
 
-  send(msgBytes: ArrayBufferView) {
+  send(msgBytes: Uint8Array) {
```

Comment:
This makes sense as `fetch` now only accepts `Uint8Array` and not `ArrayBufferView`. Can you replace `ArrayBufferView` with `Uint8Array` across the codebase in this change please?